bookmarks/services/website_loader.py

```diff
@@ -33,5 +33,8 @@ def load_website_metadata(url: str):
 
 
 def load_page(url: str):
-    r = requests.get(url)
-    return r.text
+    try:
+        r = requests.get(url, timeout=3.0)
```

Comment:
Timeout is too low IMO, assuming that the above means 3 seconds. Ideally it would be configurable, until that is possible I would at least suggest 10 seconds. Also just to confirm: is the `timeout` parameter really specified in seconds?

Reply:
I am sorry for my impatience :)
timeout=3.0 means "3 sec timeout"
I'm not familiar with Web technology, so I'm not sure reasonable timeout second.  I think 10 seconds is also good, because
we can avoid "never-end-scraping"